src/vs/platform/extensionManagement/common/extensionManagementCLI.ts

```diff
@@ -172,6 +172,47 @@ export class ExtensionManagementCLI {
 		}
 	}
 
+	public async updateExtensions(profileLocation?: URI): Promise<void> {
+		const installedExtensions = await this.extensionManagementService.getInstalled(ExtensionType.User, profileLocation);
+
+		const installedExtensionsQuery: IExtensionInfo[] = [];
+		for (const extension of installedExtensions) {
+			if (!!extension.identifier.uuid) { // No need to check new version for an unpublished extension
+				installedExtensionsQuery.push({ ...extension.identifier, preRelease: extension.preRelease });
+			}
+		}
+
+		this.logger.trace(localize('updateExtensionsQuery', "Fetching latest versions for {0} extensions", installedExtensionsQuery.length));
+		const availableVersions = await this.extensionGalleryService.getExtensions(installedExtensionsQuery, { compatible: true }, CancellationToken.None);
+
+		const extensionsToUpdate: InstallExtensionInfo[] = [];
+		for (const newVersion of availableVersions) {
+			for (const oldVersion of installedExtensions) {
+				if (areSameExtensions(oldVersion.identifier, newVersion.identifier) && gt(newVersion.version, oldVersion.manifest.version)) {
+					extensionsToUpdate.push({
+						extension: newVersion,
+						options: { operation: InstallOperation.Update, installPreReleaseVersion: oldVersion.isPreReleaseVersion }
+					});
+				}
+			}
+		}
+
+		if (!extensionsToUpdate.length) {
+			this.logger.info(localize('updateExtensionsNoExtensions', "No extension to update"));
+			return;
+		}
+
+		this.logger.info(localize('updateExtensionsNewVersionsAvailable', "Updating extensions: {0}", extensionsToUpdate.map(ext => ext.extension.identifier.id).join(', ')));
+		const installationResult = await this.extensionManagementService.installGalleryExtensions(extensionsToUpdate);
+
+		const failed: string[] = installationResult.filter(ext => ext.error).map(ext => ext.identifier.id);
+		if (failed.length) {
+			throw new Error(localize('updateExtensionsFailed', "Failed updating extensions: {0}", failed.join(', ')));
+		}
+
+		this.logger.info(localize('updateExtensionsDone', "Successfully updated {0} extensions", extensionsToUpdate.length));
```

Comment:
```suggestion
	for (const extensionResult of installationResult) {
		if (extensionResult.error) {
			this.logger.error(localize('errorUpdatingExtension', "Error while updating extension {0}: {1}", extensionResult.identifier.id, getErrorMessage(extensionResult.error)));
		} else {
			this.logger.info(localize('successUpdate', "Extension '{0}' v{1} was successfully updated.", extensionResult.identifier.id, extensionResult.local.manifest.version));
		}
	}
```

Reply:
Ok, just changed `extensionResult.local.manifest.version` to `extensionResult.local?.manifest.version` while commiting suggestion (probably a typo, avoids the error : `'extensionResult.local' is possibly 'undefined'.ts(18048)` )